tox.ini

```diff
@@ -12,6 +12,6 @@ install_command = pip install -U {opts} {packages}
 setenv = TOX_ENV_NAME={envname}
 passenv = TOX_* TRANSIFEX_USER TRANSIFEX_TOKEN TRANSIFEX_PROJECT CI CI_* CIRCLECI CIRCLE* APPVEYOR* TERM*
 commands = python -V
-           coverage run --source txclib setup.py test
+           coverage run setup.py test
            bash ./contrib/test_build.sh
-           bash -c 'test -n "$CI" && codecov -e TOX_ENV_NAME || coverage report'
+           bash -c 'test -n "$CI" && codecov -e TOX_ENV_NAME'
```

Comment:
@deathbird sorry I was trying to make it do something useful locally. I was trying to make it upload with `codecov` when run via the CI or run `coverage report` locally (to print out the coverage report to the terminal).

If this is causing issues you may want to return this to  `codecov -e TOX_ENV_NAME` which was how it was before my changes.

Reply:
@terencehonles let me first say a big thank you for your contribution.

The reasons I did minor changes in your commit were:
1. the `coverage run --source...` command for some reason affected negatively the coverage percentage and subsequently this required check failed on the CI. Haven't had the time to research it extensively, your suggestion to use the --source seems correct to me, however to proceed with the PR I reverted it to the previous command.
2. I removed the `...|| coverage report` of the last command so that the command doesn't retrun success if the codecov part of the command fails.

Reply:
ah, sounds like I should have used an if/else so it didn't squelch the error!

`if [[ -n "$CI" ]]; then codecov -e TOX_ENV_NAME; else coverage report; fi` would have been more appropriate.